client/layout/sidebar/index.jsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+export default React.createClass( {
+	displayName: 'Sidebar',
+
+	propTypes: {
+		className: React.PropTypes.string,
```

Comment:
Not used. [Related?](https://github.com/Automattic/wp-calypso/pull/2272/files#r49600502)